name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SelfWealthPDFExtractor.java

```diff
@@ -20,14 +20,15 @@ public SelfWealthPDFExtractor(Client client)
         super(client);
 
         addBankIdentifier("SelfWealth"); //$NON-NLS-1$
+        addBankIdentifier("Selfwealth"); //$NON-NLS-1$
 
         addBuySellTransaction();
     }
 
     @Override
     public String getLabel()
     {
-        return "SelfWealth"; //$NON-NLS-1$
+        return "Selfwealth Limited ABN"; //$NON-NLS-1$
```

Comment:
I don't entirely understand the code but to be clear:
> Business Name: SelfWealth
> Entity name: SELFWEALTH LTD
> https://abr.business.gov.au/ABN/View?abn=52154324428

Reply:
done